### PR TITLE
OBPIH-5076 Add missing translation labels on stock movement

### PR DESCRIPTION
**Pull request by drodzewicz**

### CODE REVIEW

**Review by awalkowiak — COMMENTED** [review]
@jmiranda do we want to go with `g:message` -> `warehouse:message` change, or we're waiting with the `g:message` investigation?

**Comment by jmiranda** [comment]
> @jmiranda do we want to go with `g:message` -> `warehouse:message` change, or we're waiting with the `g:message` investigation?

I would prefer to wait on the investigation so we don't needlessly change this all around.

**Comment by drodzewicz** [comment]
I have rolled back my changes where I changed `g:message` into `warehouse:message` since we have kind of figured out why it wasn't translating labels.

File: grails-app/views/returns/show.gsp [hunk 1]
```diff
@@ -130,7 +130,12 @@
                                 <g:message code="stockMovement.status.label"/>
                             </td>
                             <td class="value">
-                                <format:metadata obj="${stockMovement?.shipment?.status?:stockMovement?.statusCode}"/>
+                                <g:if test="${stockMovement?.shipment?.status}">
```
Comment (awalkowiak) [inline grails-app/views/returns/show.gsp:133]: @drodzewicz Is there a reason behind this change into `if-else`? You could add the `?.code` to the first part and have the same result:
```<format:metadata obj="${stockMovement?.shipment?.status?.code ?: stockMovement?.statusCode}"/>```

File: grails-app/views/stockMovement/show.gsp [hunk 2]
```diff
@@ -166,7 +166,12 @@
                                 <warehouse:message code="stockMovement.status.label"/>
                             </td>
                             <td class="value">
-                                <format:metadata obj="${stockMovement?.shipment?.status?:stockMovement?.requisition?.status }"/>
+                                <g:if test="${stockMovement?.shipment?.status}">
```
Comment (awalkowiak) [inline grails-app/views/stockMovement/show.gsp:169]: Same question as above

**Pull request merged**